spec/features/activities/work_package/activities_spec.rb

```diff
@@ -980,18 +980,224 @@
       # we need to wait a bit before triggering the update below
       # otherwise the update is already picked up by the initial (async) workpackage attributes update called in the connect hook
       # and we wouldn't test the polling based update below
-      sleep 2
+      sleep 2 # rubocop:disable OpenProject/NoSleepInFeatureSpecs
       wp_page.expect_attributes(subject: work_package.subject) # check if the initial update picked up the original subject
 
       # simulate another user is updating the work package subject
       # this btw does behave very strangely in test env and will not assign the change to the specified user
-      WorkPackages::UpdateService.new(user: admin, model: work_package).call(subject: "Subject updated")
+      WorkPackages::UpdateService.new(user: member, model: work_package).call(subject: "Subject updated")
+      work_package.journals.reload.last.update!(user: member) # manually set the user to member
 
       # activity tab should show the updated attribute
       activity_tab.expect_journal_changed_attribute(text: "Subject updated")
 
       # work package page should also show the updated attribute
       wp_page.expect_attributes(subject: "Subject updated")
     end
+
+    it "shows the updated work package attribute without reload after switching back to the activity tab", :aggregate_failures do
+      # wait for the latest comments to be loaded before proceeding!
+      activity_tab.expect_journal_notes(text: "First comment by member")
+      wp_page.expect_attributes(subject: work_package.subject)
```

Comment:
🟠 Q: Seems we're calling `wp_page.expect_attributes(subject: work_package.subject)` twice - can we replace it with a single wait_for statement?

```
wait_for { page }.to have_css(".inline-edit--container.subject", text: work_package.subject)
```

perhaps in this case, inline assertions are more concise